src/main/java/org/jabref/gui/push/PushToSublimeText.java

```diff
@@ -0,0 +1,86 @@
+package org.jabref.gui.push;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.JabRefExecutorService;
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.gui.icon.JabRefIcon;
+import org.jabref.gui.util.StreamGobbler;
+import org.jabref.logic.util.OS;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.preferences.PreferencesService;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class for pushing entries into SublimeText.
+ */
+public class PushToSublimeText extends AbstractPushToApplication {
+
+    public static final String NAME = PushToApplications.SUBLIME_TEXT;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PushToSublimeText.class);
+
+    public PushToSublimeText(DialogService dialogService, PreferencesService preferencesService) {
+        super(dialogService, preferencesService);
+    }
+
+    @Override
+    public String getDisplayName() {
+        return NAME;
+    }
+
+    @Override
+    public JabRefIcon getApplicationIcon() {
+        return IconTheme.JabRefIcons.APPLICATION_SUBLIMETEXT;
+    }
+
+    @Override
+    public void pushEntries(BibDatabaseContext database, List<BibEntry> entries, String keyString) {
+        couldNotConnect = false;
+        couldNotCall = false;
+        notDefined = false;
+
+        commandPath = preferencesService.getPushToApplicationPreferences().getCommandPaths().get(this.getDisplayName());
+
+        // Check if a path to the command has been specified
+        if ((commandPath == null) || commandPath.trim().isEmpty()) {
```

Comment:
Gibt's da nicht was von ~~Ratiopharm~~ StringUtils?

Reply:
This is just a copy paste of the first part of the execution method from  the AbstractPushToApplication